packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -60,27 +61,27 @@ export class ChoiceGroup extends BaseComponent<IChoiceGroupProps, IChoiceGroupSt
       // Need to assign role application on containing div because JAWS doesnt call OnKeyDown without this role
       <div role='application' className={ className }>
         <div
-          className='ms-ChoiceFieldGroup'
+          className={ css('ms-ChoiceFieldGroup', styles.root) }
```

Comment:
We can do more cleanup in the future, but we're inconsistent here. The component is `ChoiceGroup` and then we have classes like `ms-ChoiceFieldGroup` and `ms-ChoiceField`?